GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -316,6 +327,12 @@ public void Add(GitRevision revision, RevisionNodeFlags types = RevisionNodeFlag
         {
             _graphModel.Add(revision, types);
 
+            if (ToBeSelectedObjectIds.Contains(revision.ObjectId))
+            {
+                ToBeSelectedObjectIds.Remove(revision.ObjectId);
```

Comment:
I believe this returns `bool`, so you can avoid the double lookup (i.e. remove call to `Contains`).

Reply:
Fixed directly in master